src/Uno.UI/UI/Xaml/Controls/TextBox/TextBox.wasm.cs

```diff
@@ -44,6 +44,18 @@ private void UpdateTextBoxView()
 			}
 		}
 
+		partial void OnSelectionChangedPartial()
+		{
+			if (Text is { Length: > 0 } text || string.IsNullOrEmpty(Text) || SelectionStart + SelectionLength > Text.Length)
```

Comment:
> Text is { Length: > 0 } text || string.IsNullOrEmpty(Text)

I'm confused. Isn't this always true?

Reply:
Because this is always true, I'm suggesting in the below comments to revert those changes and keep the always-true branch in the language-agnostic TextBox.cs